docs/admin_architecture.rst

```diff
@@ -0,0 +1,31 @@
+The Admin with Versioning
+==========================
+
+
+The content model admin
+------------------------
+Versioning modifies (monkeypatches) the admin for each :term:`content model <content model>`. This is because
+versioning duplicates content model records every time a new version is created (since content models hold the version data
+that's content type specific). Versioning therefore needs to limit the queryset in the content model admin to
+include only the records for the latest versions.
+
+
+The Version model admin
+------------------------
+
+Proxy models
++++++++++++++
+Versioning generates a `proxy model
+<https://docs.djangoproject.com/en/dev/topics/db/models/#proxy-models>`_ of Version
```

Comment:
Can you refer to `:class:djangocms_versioning.models.Version` here? Think it makes sense in versioning, as this documentation gets quite complex